### ISSUE TITLE
VMWare: v2.3.4 Ubuntu 14.04.1 boxes cannot mount shared folders

### ISSUE THREAD

**Issue by martinisoft** [chef/bento#785]
It looks like something may have broken between the 8.5.5/8.5.6 release of VMWare Fusion and Ubuntu 14.04 boxes. When I try to launch these boxes within Test Kitchen I get the following error:

```
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Failed to complete #create action: [Expected process to exit with [0], but received '1'
---- Begin output of vagrant up --no-provision --provider vmware_fusion ----
STDOUT: Bringing machine 'default' up with 'vmware_fusion' provider...
==> default: Cloning VMware VM: 'bento/ubuntu-14.04'. This can take some time...
==> default: Checking if box 'bento/ubuntu-14.04' is up to date...
==> default: Verifying vmnet devices are healthy...
==> default: Preparing network adapters...
==> default: Fixed port collision for 22 => 2222. Now on port 2206.
==> default: Starting the VMware VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 192.168.223.129:22
    default: SSH username: vagrant
    default: SSH auth method: private key
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default:
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Forwarding ports...
    default: -- 22 => 2206
==> default: Setting hostname...
==> default: Configuring network adapters within the VM...
==> default: Waiting for HGFS to become available...
==> default: Enabling and configuring shared folders...
STDERR: WARNING: The VMX file for this box contains a setting that is automatically overwritten by Vagrant
WARNING: when started. Vagrant will stop overwriting this setting in an upcoming release which may
WARNING: prevent proper networking setup. Below is the detected VMX setting:
WARNING:
WARNING:   ethernet0.pcislotnumber = "33"
WARNING:
WARNING: If networking fails to properly configure, it may require this VMX setting. It can be manually
WARNING: applied via the Vagrantfile:
WARNING:
WARNING:   Vagrant.configure(2) do |config|
WARNING:     config.vm.provider :vmare_fusion do |vmware|
WARNING:       vmware.vmx["ethernet0.pcislotnumber"] = "33"
WARNING:     end
WARNING:   end
An error occurred while executing `vmrun`, a utility for controlling
VMware machines. The command and output are below:

Command: ["enableSharedFolders", "/Users/martinisoft/dnsimple/chef-dnsimple/.kitchen/kitchen-vagrant/kitchen-chef-dnsimple-create-record-ubuntu-1404/.vagrant/machines/default/vmware_fusion/1e11d750-a1fc-4676-b360-6837731b92a7/ubuntu-14.04-amd64.vmx", {:notify=>[:stdout, :stderr]}]

Stdout: Error: There was an error mounting the Shared Folders file system inside the guest operating system

Stderr:
---- End output of vagrant up --no-provision --provider vmware_fusion ----
Ran vagrant up --no-provision --provider vmware_fusion returned 1] on create-record-ubuntu-1404
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

I have resolved this locally by re-building the 14.04 box against VMWare Fusion 8.5.6 which has done the trick for me locally. VMWare may have altered the tools due to some CVE changes in the 8.5.6 release. So I have a `kitchen.local.yml` config using my locally built box for now until an update can be built and shipped.

**Comment by rmoriz** [comment]
This happens to me with Fusion 8.5.6 and the current `debian-8.7`…

**Comment by rmoriz** [comment]
Last time this happend, the issue was caused by using an old version of fusion to build the boxes. Maybe VMware again changed or fixed something which broke it again?

**Comment by cheeseplus** [comment]
This should have been resolved in the latter 2.3.x's so please open a new issue if not.
